src/lib/y2partitioner/confirm_recursive_delete.rb

```diff
@@ -39,7 +39,7 @@ def confirm_recursive_delete(devices, headline, label_before, label_after)
       button_box = ButtonBox(
         PushButton(Id(:yes), Opt(:okButton), recursive_delete_yes_label),
         PushButton(
-          Id(:no_button),
+          Id(:no),
           Opt(:default, :cancelButton),
```

Comment:
ugh, there are no cancelButton, not?

Reply:
ah, now I get options, it is not cancel as cancel but to pretend cancel behavior :angry: